packages/cli/src/modules/insights/__tests__/insights.service.test.ts

```diff
@@ -664,3 +670,93 @@ describe('getMaxAgeInDaysAndGranularity', () => {
 		});
 	});
 });
+
+describe('shutdown', () => {
+	let insightsService: InsightsService;
+
+	const mockCollectionService = mock<InsightsCollectionService>({
+		shutdown: jest.fn().mockResolvedValue(undefined),
+		stopFlushingTimer: jest.fn(),
+	});
```

Comment:
There are various below that we can also mock like this to avoid `as unknown as`.